index.bs

```diff
@@ -49,16 +49,25 @@ url: https://drafts.csswg.org/css-transitions-1/#transitions; type: dfn; text: C
 
 <h2 id="basic-principles">Principles behind design of Web APIs</h2>
 
-<h3 id="priority-of-constituencies">Priority of constituencies</h3>
-
-In case of conflict,
-consider users over authors over implementors over specifiers over theoretical purity.
-In other words costs or difficulties to the user
-should be given more weight than costs to authors;
-which in turn should be given more weight than costs to implementors;
-which should be given more weight than costs to authors of the specification itself,
-which should be given more weight than those proposing changes for theoretical reasons alone.
-Of course, it is preferred to make things better for multiple constituencies at once.
+<h3 id="priority-of-constituencies">Put user needs first (Priority of Constituencies)</h3>
+
+If a trade-off needs to be made,
+always put user needs above all.
```

Comment:
I think it might be worth trying to explain *why* user needs are most important.  From the initial comment, part of it is that they're the largest number, but I think that's not all of it, and we should try to explain why if we can figure out how to.

Reply:
Made an attempt at this.